src/_includes/components/meta-title.webc

```diff
@@ -1,9 +1,9 @@
 <script webc:type="js">
 	let text = $data.site.title;
 
-	if (title) {
-		text = `${title} « ` + text;
-	}
+	if ($data.title) {
+    text = $data.title + ` « ` + text;
+  }
```

Comment:
Can you fix the whitespace here and I’ll merge the PR; should be tabs, not spaces. Thanks.